### ISSUE TITLE
After reboot unbound service starts after IPsec service wich causes permanent connection error when the ipsec remote gateway is a DNS name

### ISSUE THREAD

**Issue by dstr-wpd** [opnsense/core#7767]
**Important notices**

Our forum is located at https://forum.opnsense.org , please consider joining discussions there in stead of using GitHub for these matters.

Before you ask a new question, we ask you kindly to acknowledge the following:

- [ x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x ] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

After reboot unbound service starts after IPsec service wich causes permanent connection error when the ipsec remote gateway is a DNS name

This is a behaivor since upgrade to 24.7 major relase. logs attached:

[system.log](https://github.com/user-attachments/files/16596006/system.log)
![Screenshot 2024-08-13 093922](https://github.com/user-attachments/assets/68348be6-ce50-4a82-b3d3-4a2c2efbc583)
![image](https://github.com/user-attachments/assets/19869926-8812-47f0-98ba-eb6bf0265308)
keyretries is set to 0, so it should try infinite times

![image](https://github.com/user-attachments/assets/614437c4-1215-47cb-858b-818c86832d48)

is there a way to delay ipsec service or priorize unbound?

**Comment by fichtner** [comment]
Ok, I'll bite:

https://github.com/opnsense/core/blob/10aa7878cf5e49c2125d8752c20ca6dea048c1de/src/etc/rc.bootup#L98

https://github.com/opnsense/core/blob/10aa7878cf5e49c2125d8752c20ca6dea048c1de/src/etc/rc.bootup#L102

**Comment by dstr-wpd** [comment]
Do I have to change these values?

**Comment by fichtner** [comment]
No. DNS is started before IPsec. I'm not sure what your issue is.

**Comment by dstr-wpd** [comment]
2024-08-12T14:53:06	Informational	unbound	 [62342:0] info: start of service (unbound 1.20.0).
2024-08-12T14:53:06	Notice	unbound	 [62342:0] notice: init module 2: iterator
2024-08-12T14:53:06	Notice	unbound	 [62342:0] notice: init module 1: validator
2024-08-12T14:53:06	Notice	unbound	 [62342:0] notice: init module 0: python
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 3: requestlist max 0 avg 0 exceeded 0 jostled 0
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 3: 0 queries, 0 answers from cache, 0 recursions, 0 prefetch, 0 rejected by ip ratelimiting
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 2: requestlist max 0 avg 0 exceeded 0 jostled 0
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 2: 0 queries, 0 answers from cache, 0 recursions, 0 prefetch, 0 rejected by ip ratelimiting
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 1: requestlist max 0 avg 0 exceeded 0 jostled 0
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 1: 0 queries, 0 answers from cache, 0 recursions, 0 prefetch, 0 rejected by ip ratelimiting
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 0: requestlist max 0 avg 0 exceeded 0 jostled 0
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: server stats for thread 0: 0 queries, 0 answers from cache, 0 recursions, 0 prefetch, 0 rejected by ip ratelimiting
2024-08-12T14:53:03	Informational	unbound	 [44853:0] info: service stopped (unbound 1.20.0).
2024-08-12T14:53:01	Informational	unbound	 [44853:0] info: start of service (unbound 1.20.0).



looks like its been started twice
will try another log lever
